### DISCUSSION TITLE
Increase brightness of DSS survey

### DISCUSSION THREAD

**Discussion by HowDidYouKillMe** [Stellarium/stellarium#2928] [Q&A] [answered]
Is there an option to increase the brightness of the sky? Many targets in the survey seem too dark. 

Thanks in advance

**Comment by gzotti** [comment, marked as answer]
There is no such option. Just take in mind that DSS was created from scanning photographic glass plates of an all-sky survey from the 1990s and cannot be compared to multi-hour exposures of just the highlights on modern CCDs followed by advanced digital image processing.
